app/assets/javascripts/application.js

```diff
@@ -10,13 +10,20 @@
 // Read Sprockets README (https://github.com/rails/sprockets#sprockets-directives) for details
 // about supported directives.
 //
+//= require jquery
 //= require rails-ujs
 //= require_tree .
-
-//= require jquery
 // GOV.UK Frontend
 // https://github.com/alphagov/govuk-frontend/blob/master/docs/installation/installing-with-npm.md
 //
 //= require govuk-frontend/all
 
 //= require moj
+
+
+$(document).ready(function() {
```

Comment:
We are already requiring and initialising the modules in the above line `//= require moj`
It uses the already present `moj.js` file. Do we need again this?

Reply:
yes we need to initialise `moj`